packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -51,6 +51,19 @@ import {
 const SR_BACKEND_FAILURE_LINK =
   'https://xen-orchestra.com/docs/backup_troubleshooting.html#srbackendfailure44-insufficient-space'
 
+const ThinProvisionedTip = ({ label }) => (
+  <Tooltip content={_(label)}>
+    <a
+      className='text-info'
+      href={SR_BACKEND_FAILURE_LINK}
+      rel='noopener noreferrer'
```

Comment:
Why do you need this when we never used it before?

Reply:
It's used for more [security](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/). We also used [here](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/xo-app/logs/log-alert-body.js#L325-L330).